packages/js/api/package.json

```diff
@@ -45,14 +45,14 @@
 	},
 	"devDependencies": {
 		"@types/create-hmac": "1.1.0",
-		"@types/jest": "^27.0.2",
+		"@types/jest": "^27.4.1",
 		"@types/node": "13.13.5",
 		"@typescript-eslint/eslint-plugin": "^5.3.1",
 		"@typescript-eslint/parser": "^5.3.1",
 		"axios-mock-adapter": "^1.20.0",
 		"eslint": "^8.2.0",
-		"jest": "^25",
-		"ts-jest": "^25",
+		"jest": "^27.5.1",
+		"ts-jest": "^27",
```

Comment:
Any reason we're not syncing ts-jest as well? with growing TS adoption it seems like we might as well get in front of that and add it to syncpack now?

Reply:
I didn't add it to limit the scope of this PR, but if you say so, I'd add it as well.